File: checAlphabet/alphabet.c
#include <stdio.h>

int main() {
 char c;
 printf("Enter A Character:");
 scanf("%c",&c);
 printf("Your Input  Value is: %c",c);
 
 if((c>='a' && c<='z')||(c>='A' && c<='Z')){
     printf("\nis a alphabet");
 }
 else{
     printf("is not a alphabet");
 }
 

    return 0;
}

File: checAlphabet/alphabet.js
const inputValue = prompt("Write Your balue:")
console.log("your value is ", inputValue)
if ((inputValue >= 'a' && inputValue <= 'z') || (inputValue >= 'A' && inputValue <= 'Z')) {
    console.log('value is alphabet');
}
else {
    console.log('value is not a alphabet');
}